apps/main/src/lend/components/ChartOhlcWrapper/TradesData.tsx

```diff
@@ -17,21 +17,21 @@ const TradesData: React.FC<TradesDataProps> = ({ lendTradesData, chainId }) => {
   return (
     <>
       {lendTradesData.map((transaction, index) => (
-        <TransactionRow key={`${transaction.transaction_hash}-${transaction.sold_id}-trade-${index}`}>
-          <Event href={networks[chainId].scanTxPath(transaction.transaction_hash)} rel="noopener" target="_blank">
+        <TransactionRow key={`${transaction.txHash}-${transaction.idSold}-trade-${index}`}>
```

Comment:
why do we need to abbreviate?
```suggestion
        <TransactionRow key={`${transaction.hash}-${transaction.idSold}-trade-${index}`}>
```
or
```suggestion
        <TransactionRow key={`${transaction.transactionHash}-${transaction.idSold}-trade-${index}`}>
```

Reply:
txHash is the convention as hashes can be something else, and 'transaction' is long and cumbersome in this context. It seems 'txHash' is also already used in other places of the code. By the way, I should rather rename 'transaction' to 'trade' as that's a better and more specific term than the generic 'transaction'.